src/include/migraphx/op/convert.hpp

```diff
@@ -20,11 +20,14 @@ namespace op {
 struct convert : unary<convert>
 {
     shape::type_t target_type = shape::half_type;
+    float scale               = 1.0f;
+    float shift               = 0.0f;
```

Comment:
Why is scale and shift added here? And these should be arguments since they dont affect the shape.

Reply:
removed the scale and shift.